src/login/login.jsx

```diff
@@ -25,49 +25,59 @@ const Login = () => {
   };
 
   return (
-    <div className="loginContainer">
-      <div className="center">
-        {userLoggedIn && <Navigate to={"/home"} replace={true} />}
-        <div>
-          <h1>Welcome Back</h1>
-        </div>
-        <form onSubmit={onSubmit}>
-          <div className="txtField">
-            <input
-              type="text"
-              autoComplete="email"
-              required
-              value={email}
-              onChange={(e) => {
-                setEmail(e.target.value);
-              }}
-            />
-            <span></span>
-            <label>Email</label>
-          </div>
+    <div>
```

Comment:
This div is not used to enforce style anymore, perhaps it can be removed or replaced with a fragment (`<></>`)